src/widgets/toggle-refinement/__tests__/toggle-refinement-test.js

```diff
@@ -332,6 +332,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/toggle-refi
           addDisjunctiveFacetRefinement: jest.fn(),
           search: jest.fn(),
           setPage: jest.fn(),
+          getIndex: jest.fn(),
```

Comment:
@eunjae-lee I add to add this after merging the `createSendEvent` function, could you please check?

Reply:
Can you explain why you had to add this? With the previous helper initialization, it didn't work?

Reply:
The `getIndex()` function that you was `undefined` in the tests
https://github.com/algolia/instantsearch.js/blob/f5b796c78e4f208f9daea152239d6bdd2477fe93/src/connectors/toggle-refinement/connectToggleRefinement.js#L37

Reply:
Yeah, but in this file, `createSendEvent` function is not tested, and it's tested in `connectToggleRefinement-test.js`.

If you revert this part back to 

```js
        helper = jsHelper({}, '');
        helper.removeDisjunctiveFacetRefinement = jest.fn();
        helper.addDisjunctiveFacetRefinement = jest.fn();
        helper.search = jest.fn();
```

the tests still pass, right?

Reply:
You're right, all the tests still pass. I might have changed something while I was trying to resolve the conflicts.